modules/rosa-sts-understanding-ocm-role.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// rosa_architecture/rosa-sts-about-iam-resources.adoc
+//
+:_content-type: CONCEPT
+[id="rosa-sts-understanding-ocm-role_{context}"]
+= Understanding the {cluster-manager} role
+
+Creating ROSA clusters in {cluster-manager-url} requires an `ocm-role` IAM role. The basic `ocm-role` IAM role permissions allow you to perform cluster maintenance within {cluster-manager}. If you need elevated permissions, you can add the `--admin` option to the `rosa create` command to create an `ocm-role` resource with additional permissions needed for administrative tasks.
```

Comment:
In reading the "If you need elevated permissions..." it makes me think "no i dont".  Its almost a knee-jerk reaction.  Can we find a more seamless way to show customers the "easy-way" so that it is not split between parts (like this regular doc and the note below?  Also, we could change the verbiage to not say "if you need", but rather in explaining the "easy way",  to "In order to automatically create the operator roles and OIDC provider please add `--admin` so that the ocm-role is created with additional permissions in order to complete the tasks".   Or something like that.